### ISSUE TITLE
No Unicode support?

### ISSUE THREAD

**Issue by davidebeatrici** [crypto2011/IDR#77]
![](https://user-images.githubusercontent.com/5897523/177694119-4c25c71f-0887-4744-858e-a00e560bc975.png)

[OpenIV_4.1_1502_2022-01-21.zip](https://github.com/crypto2011/IDR/files/9060493/OpenIV_4.1_1502_2022-01-21.zip)

**Comment by sarog** [comment]
Yup, unfortunately IDR lacks [Unicode support](https://github.com/crypto2011/IDR/issues/56#issuecomment-610479417) for some time now.

**Comment by davidebeatrici** [comment]
That's unfortunate. @crypto2011 Is it something you're perhaps already working on?
